packages/contracts-bedrock/src/dispute/FaultDisputeGame.sol

```diff
@@ -68,19 +78,24 @@ contract FaultDisputeGame is IFaultDisputeGame, Clone, Semver {
     /// @param _vm An onchain VM that performs single instruction steps on a fault proof program
     ///            trace.
     /// @param _l2oo The trusted L2OutputOracle contract.
-    /// @custom:semver 0.0.4
+    /// @param _blockOracle The block oracle, used for loading block hashes further back
+    ///                     than the `BLOCKHASH` opcode allows as well as their estimated
+    ///                     timestamps.
+    /// @custom:semver 0.0.6
     constructor(
         Claim _absolutePrestate,
         uint256 _maxGameDepth,
         Duration _gameDuration,
         IBigStepper _vm,
-        L2OutputOracle _l2oo
-    ) Semver(0, 0, 4) {
+        L2OutputOracle _l2oo,
+        BlockOracle _blockOracle
+    ) Semver(0, 0, 6) {
```

Comment:
This bumps 2 versions but isn't a big deal

Reply:
Since its pre major v1

Reply:
This was intentional - squashed the stack due to overlap, but there were 2 versions worth of changes here.

Reply:
it feels to me like versions should be bumped only when the changes are merged into trunk, otherwise you get this weird version gap in trunk.

But again not a big deal, just shouldn't do in the future imo.